### PR TITLE
Adding Hetzner Cloud CLI (hcloud command)

### PR DESCRIPTION
**Pull request by confund0**

### CODE REVIEW

File: packages/hcloud/build.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/hetznercloud/cli
+TERMUX_PKG_DESCRIPTION="Hetzner Cloud command line client"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="1.30.1"
+HCLOUD_PKG_VERSION="v${TERMUX_PKG_VERSION}"
```
Comment (Grimler91) [inline packages/hcloud/build.sh:6]: Please drop this variable and just specify `v${TERMUX_PKG_VERSION}` in SRCURL and `LD_FLAGS`

**Review by Grimler91 — CHANGES_REQUESTED** [review]
Thanks for the PR! Please see comment

**Comment by confund0** [comment]
Done. New PR 11376.

**Comment by 2096779623** [comment]
> Done. New PR 11376.

Don't create multiple PRs for the same request.
Can you squash the commit?

**Pull request closed without merging**